README.md

```diff
@@ -245,3 +245,7 @@ Below are some tip(s) about error(s) that may occurs, (more will be added later
   
   So in this case, if you look the the output container path, you will find a __migrate blob that is locked via lease which you'll need to break the lease of prior to another run that writes to this location.  So if you know that another tool is not working on this output path, then you will need to break the lease the '__migrate' blob.  This can be done manually using the azure portal or azure storage explorer.  Another approach is to run the same AMSMigrate 'asset' command that writes to this output path but append '--break-output-lease' flag which will automatically break the lease of the __migrate blob for you.
 
+# POST-MIGRATION
+Following MAS retirement, you can proceed with asset migration from a storage account using the "storage" command. For additional information, please see the "..\doc\storageCommand.md" document.
```

Comment:
Typo MAS -->  AMS.   

Technically customer can run "storage" command even before AMS is shutdown, as long as they can pass in the correct storage account name, and this storage account name is not the same as the AMS account name, which is the case for most of our customers.